abce/__init__.py

```diff
@@ -535,10 +535,8 @@ def delete_agent(self, name, quite=True):
         group, id = name
 
         pg = self._processor_groups[id % self.processes]
-        if quite:
-            pg.replace_with_dead(group, id, SilentDeadAgent)
-        else:
-            pg.replace_with_dead(group, id, LoudDeadAgent)
+        self.num_of_agents_in_group[group] -= 1
+        pg.delete_agent(group, id)
```

Comment:
At least the death should be logged at here.

Reply:
fixed